tools/dockerfile/test/bazel/Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN pip install futures==2.2.0 enum34==1.0.4 protobuf==3.5.2.post1 six==1.10.0 t
 # Bazel installation
 RUN echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8" > /etc/apt/sources.list.d/bazel.list
 RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
-RUN apt-get -y update && apt-get -y install bazel=0.15.0 && apt-get clean
+RUN apt-get -y update && apt-get -y install bazel=0.16.1 && apt-get clean
```

Comment:
FYI (as discussed offline), this Dockerfile is only used for the jobs `grpc_bazel_build` and `grpc_python_bazel_test` to build with bazel locally (and run tests python tests with bazel locally too). It doesn't influence the Foundry builds at all.